### DISCUSSION TITLE
Resuming training

### DISCUSSION THREAD

**Discussion by protomato2** [152334H/DL-Art-School#45] [Q&A] [answered]
First, thank you for everything you do, 152334H.
I have a question.
What is exactly required to be able to resume training process? Is it possible in colab? What files there have to be and what settings?
In colab, it doesn't seem to resume training, even if training state files are present, it still starts from 0. Do the 0 state files have to be enabled too?

**Comment by 152334H** [comment, marked as answer]
See https://github.com/152334H/DL-Art-School/issues/7, resuming is not a simple checkbox in the colab atm

**Reply by protomato2** [reply]
Thank you, didn't know I have to comment out pretrained model too, able to resume now

**Comment by FurkanGozukara** [comment]
for resuming i give checkpoint pth
isnt that correct way?

**Comment by agiusthomas** [comment]
anyone figured out a way to resume?

**Reply by FurkanGozukara** [reply]
> anyone figured out a way to resume?

yes i use checkpoint in advanced tab. it is same as resume 

here my full tutorial: [Master Deep Voice Cloning in Minutes: Unleash Your Vocal Superpowers! Free and Locally on Your PC](https://youtu.be/OiMRlqcgDL0)
